tests/integration/test_valid_data_in_request.py

```diff
@@ -97,3 +99,15 @@ def test_various_packages(test_env):
         utils.assert_properly_completed_response(completed_response)
 
         assert len(completed_response.data["dependencies"]) == package["dependencies_count"]
+
+
+def test_utc_datetime(test_env):
+    env_data = utils.load_test_data("pip_packages.yaml")["local_path"]
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
+    expected = datetime.datetime.now(timezone.utc).isoformat()[:16]
+    initial_response = client.create_new_request(
+        payload={"repo": env_data["repo"], "ref": env_data["ref"]}
+    )
+    completed_response = client.wait_for_complete_request(initial_response)
+    actual = completed_response.data["created"][:16]
```

Comment:
Isn't it fun to test things involving time :grin: 
I wonder if we might see test failures when time is truncated to the minute and the expected/actual fall on opposite sides of it. @lkolacek what do you think?

Reply:
We can always add some threshold :) if there's more than 10s difference it fails

Reply:
Yeah I agree, this was something I was worried about. I will apply the threshold on both the integration and unit tests.